src/styles/nav.scss

```diff
@@ -0,0 +1,53 @@
+.nav {
+  &__ul {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style: none;
+  }
+
+  &__item {
+    margin-right: 20px;
+    height: $nav-item-height;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+
+    &-link {
+      padding: (($nav-item-height - $nav-line-height) / 2) 0;
+      font-weight: 500;
+      font-size: 12px;
+      font-style: normal;
+      line-height: $nav-line-height;
+      color: $main-accent-color;
+      text-decoration: none;
+      text-transform: uppercase;
+      transition: color 0.5s;
+    }
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+    &-link:hover {
+      color: $blue-accent-color;
+    }
+  }
+}
+
+.is-active {
```

Comment:
Let's transform it to link modifier
So, move all styles to modifier class in css, and leave `is-active` class only in html for tests passing